frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -214,6 +214,21 @@ class Worksheet extends React.Component {
         return wsItemsIdDict;
     }
 
+    convertDatetimeToLocalDatetime = (date) => {
```

Comment:
Let's use the native `Date` functions for this instead.

Can you see if you can get the backend to return the date in ISO 8601 format (like "2020-11-18T06:43:05Z")?

Then, we can just do this:
```
date = new Date("2020-11-18T06:43:05Z");
return date.toString();
```

Reply:
I think the difference is the current method does the format processing in the frontend while your purposed method does the processing in the backend? Why do we want to do the processing in the backend?

Reply:
Change applied.